proxy/fadeintesting_test.go

```diff
@@ -270,7 +270,7 @@ func (p *fadeInProxy) addInstances(n int) {
 			DataClients:    []routing.DataClient{client},
 			PostProcessors: []routing.PostProcessor{
 				loadbalancer.NewAlgorithmProvider(),
-				fadein.NewPostProcessor(),
+				fadein.NewPostProcessorWithOptions(fadein.Options{}),
```

Comment:
should not this also have the endpointsregistry passed?

Reply:
However, added endpointregistry here for the sake of uniformity, not to confuse anyone and to show other developers that it's a good practice to add it.